obsbot/cogs/public/factoids.py

```diff
@@ -159,7 +163,10 @@ async def on_message(self, msg: Message):
             else:  # factoid does not exit
                 return
 
-        if not self.bot.is_supporter(msg.author) and self.limiter.is_limited(factoid_name, msg.channel.id):
+        if not self.bot.is_supporter(msg.author) and (
+            self.member_limiter.is_limited(factoid_name, msg.author.id)
```

Comment:
This would still limit by factoid per user. I don't know what the spam looks like, but if they're using different factoids in quick succession then this would not work.

Reply:
Hmm, that's a good point. I'll probably remove it then since using `None` as a factoid is janky, and the whole reason people are spamming to begin with is because they can ping spam, which my other change fixes.